src/test/resources/google/cloud/compute/v1small/compute-versioned-two-services.proto.baseline

```diff
@@ -0,0 +1,761 @@
+// Copyright 2020 Google LLC
```

Comment:
```suggestion
// Copyright 2024 Google LLC
```

Reply:
The date is derived from the Discovery revision field, so this is WAI. But to your point, since these Discovery fragments are new, I'll change them to have a 2024 date.